C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -248,15 +264,15 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
         /// <param name="activity">The updated activity in the form `{id: <id of activity to update>, ...}`</param>
         public override async Task<ResourceResponse> UpdateActivityAsync(ITurnContext turnContext, Activity activity, CancellationToken cancellationToken)
         {
-            ResourceResponse results = null;
+            dynamic results = null;
             if (activity.Id != null && activity.Conversation != null)
             {
                 try
```

Comment:
Remove the `try/catch` blocks. The Exception should be handled by the methods which call this `UpdateActivityAsync` method.